packages/semantic-tokens/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@fluentui/semantic-tokens",
+  "version": "0.0.0-alpha.1",
+  "description": "Fluent UI Theme semantic-tokens",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "./dist/index.d.ts",
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "*",
+    "@fluentui/scripts-api-extractor": "*"
+  },
+  "dependencies": {
+    "@swc/helpers": "^0.5.1",
```

Comment:
Do we actually need this?

Reply:
Not 100% certain but following the same dependency schema as our other top level packages here